BitbucketPS.psd1

```diff
@@ -99,27 +98,20 @@ PrivateData = @{
         # Tags = @()
 
         # A URL to the license for this module.
-        LicenseUri = 'https://github.com/AtlassianPS/BitbucketPS/blob/master/LICENSE'
+        # LicenseUri = ''
 
         # A URL to the main website for this project.
-        ProjectUri = 'https://github.com/AtlassianPS/BitbucketPS'
+        # ProjectUri = ''
 
         # A URL to an icon representing this module.
         # IconUri = ''
 
         # ReleaseNotes of this module
-        ReleaseNotes = 'Updating gitignore
-
-Incorporated the rest URI into the BBSession varialbe. Changed that URI to use /latest/ instead of /1.0/ (unless Atlassian tells me that''s a bad idea).  Updated output on BBCommits, BBRepositories to look a little prettier
-
-'
-
-        # External dependent modules of this module
-        # ExternalModuleDependencies = ''
+        # ReleaseNotes = ''

```

Comment:
This shall contain the link to the `CHANGELOG.md` in the root of the repo.
Feel free to create an empty (or not empty) file

Reply:
That works, I reverted so it should be back to blank.  Can be updated once this mess of a pull request is done!